en/developer_guide/smartAdmin.rst

```diff
@@ -0,0 +1,75 @@
+SmartAdmin
+==========
+
+Our back office use SmartAdmin with only some of JavaScript components (JarvisWidget and SmartNotification).
+
+SmartConfirm
+============
+
+Description
+-----------
+
+The function ``smartConfirm`` allows to display a confirmation message.
+She uses the function ``SmartMessageBox`` of component SmartNotification.
+She is used during various events like the deleting of an item (nodes, blocks, contents, ...).
+
+A confirmation message is composed of a logo, a title, a description and two buttons (Yes/No).
+
+.. image:: ../../images/smartconfirm.png
+
+The function ``smartConfirm`` takes 4 parameters:
+
+1.  the logo css class.
+2.  the message title.
+3.  the message description.
+4.  an array of callbacks functions which are called when buttons are clicked and
+the parameters of callbacks functions.
+
+The array must contain the keys ``yesCallback`` and ``NoCallback``
+which refer to respectively at the functions which manage the click event on Yes and No buttons.
+And the keys ``callBackParams`` which is an array of parameters for callbacks functions.
```

Comment:
There's no verb in that sentence + a problem "the keys [...] which are" or "the key [...] which is"